pkg/model/hvs/flavortemplate.go

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2020 Intel Corporation
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+package hvs
+
+import (
+	"github.com/google/uuid"
+)
+
+type Meta struct {
+	FlavorPart  string `json:"flavor_part,omitempty"`
+	Source      string `json:"source,omitempty"`
+	Label       string `json:"label,omitempty"`
+	Vendor      string `json:"vendor,omitempty"`
+	IPAddress   string `json:"ip_address,omitempty"`
+	BiosName    string `json:"bios_name,omitempty"`
+	BiosVersion string `json:"bios_version,omitempty"`
+	OsName      string `json:"os_name,omitempty"`
+	OsVersion   string `json:"os_version,omitempty"`
+	VmmName     string `json:"vmm_name,omitempty"`
+	VmmVersion  string `json:"vmm_version,omitempty"`
+	TpmVersion  string `json:"tpm_version,omitempty"`
+	// swagger: strfmt uuid
+	HardwareUUID *uuid.UUID `json:"hardware_uuid,omitempty"`
+	Comment      string     `json:"comment,omitempty"`
+
+	TbootInstalled bool `json:"tboot_installed,omitempty"`
+	CBNTEnabled    bool `json:"cbnt_enabled,omitempty"`
+	UEFIEnabled    bool `json:"uefi_enabled,omitempty"`
+	SUEFIEnabled   bool `json:"suefi_enabled,omitempty"`
+	BMCEnabled     bool `json:"bmc_enabled,omitempty"`
+	PFREnabled     bool `json:"pfr_enabled,omitempty"`
+
+	DigestAlgorithm string `json:"digest_algorithm,omitempty"`
+}
+
+//PCR- Tp stpre PCR index with respective PCR bank.
+type PCR struct {
```

Comment:
@mshamn6x In the future you will need to use the structures defined in this package.  Ideally, it would contain all the structures and json decoration needed to share data between the Trust-agent and HVS.